### PR TITLE
Add support for Rails 8

### PR DESCRIPTION
**Pull request by bvalentino**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -24,8 +24,10 @@ jobs:
     strategy:
       matrix:
         ruby: ["3.2", "3.3"]
-        activemodel: ["7.1"]
+        activemodel: ["7.1", "8.0"]
         include:
+          - activemodel: "8.0"
+            ruby: "3.3"
```
Comment (kakubin) [inline .github/workflows/test.yml:30]: this combination has already been tested in a matrix.

```suggestion
```

**Comment by erickguan** [comment]
Do you accept a PR extending the test to Rails 7.2 too?

**Comment by AaronLasseigne** [comment]
I'll make sure to add it.

**Comment by erickguan** [comment]
Thanks!

**Pull request merged**